torchtext/data/metrics.py

```diff
@@ -0,0 +1,93 @@
+import math
+import collections
+from torchtext.data.utils import ngrams_iterator
+
+
+def _compute_ngram_counter(tokens, max_n):
```

Comment:
It seems that this func has only one line. You may not need a func here.

Reply:
The assumption I took to keep this as a function was:
- Right now I use this function in these 2 places: https://github.com/pytorch/text/blob/c1fc7b0944e3b37d7cf05e1cf8ea3369842948b2/torchtext/data/metrics.py#L73-L75
- We might want to use it in other metrics in the future (it's a long shot as of now though)